mephisto/operations/operator.py

```diff
@@ -9,7 +9,6 @@
 import os
 import time
 import threading
-import traceback
```

Comment:
It wasn't being used globally. The file did use traceback in exceptions, but it would import it in those exceptions.

I added it back and removed the other imports of traceback in this file

Reply:
ah gotcha